src/Widgets/PagedDialog.vala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2025 elementary, Inc. <https://elementary.io>
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+/**
+ * PagedDialog is a modal {@link Gtk.Window} subclass containing an {@link Adw.NavigationView}.
```

Comment:
It would be kinda slick if Code could parse the "{@link}" pieces and make them clickable links! Would make code comments even more useful

Reply:
Yeah it's basically only useful rn if you build valadoc. Open a feature request though!